package.json

```diff
@@ -62,9 +62,12 @@
     "build:cjs": "tsc",
     "build:esm": "tsc --project tsconfig.json --module ESNext --outDir ./lib/esm",
     "build:types": "tsc --project tsconfig.json --declaration --emitDeclarationOnly --declarationMap --outDir ./lib/types",
+    "build:docker": "docker build -q -t asyncapi/modelina .",
     "test": "cross-env CI=true jest --coverage --testPathIgnorePatterns ./test/blackbox",
     "test:blackbox": "cross-env CI=true jest ./test/blackbox",
     "test:watch": "jest --watch",
+    "test:docker": "npm run build:docker && docker run asyncapi/modelina npm run test",
```

Comment:
Would `docker:test` not make more sense? As we are testing using docker, we don't test docker 🤔

Reply:
hmm... I guess it might be better to stay as is, in terms of how the rest of the scripts are 🤔

Reply:
Thoughts?

Reply:
Yeah, I'm happy with both naming patterns. At the end you could also read this as `test:*` is all related to testing. But Agree prefixing with `docker:` instead could make more sense. Changing!

Reply:
Done!